### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by tomsajuk**
Adding a PR template as discussed. I referred other PR templates to create this. Any suggestions?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## 🚀 Description
+
+<!-- Describe your changes in detail -->
+
+## 📄 Motivation and Context
+
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## 🧪 How Has This Been Tested?
```
Comment (TooAngel) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:12]: I would remove the paragraph. We have a CI server that runs the tests to make sure everything is tested.

That the tests need to be run, should be a friendly reminder, not necessarily something to describe out.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## 📦 Types of changes
+
+<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply and remove the ones not applicable -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## ✅ Checklist
```
Comment (TooAngel) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:27]: A checklist might be good, I'm not so sure about the documentation part. I guess nobody will tick any of these. But we can try it out.

**Review by TooAngel — COMMENTED** [review]
I would remove the `Testing` paragraph and I'm not sure about the documentation checklist.

Besides that I think it is a good start, I think we'll need some more iterations (I need to try it out to recognize what is missing).

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (tomsajuk) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: You are right. I removed the testing part.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 2 above)
Comment (tomsajuk) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I removed the checklist for now. We can add it later as it grows :) What say?

**Review by TooAngel — APPROVED** [review]
Looks good, thx

**Review by IanFindlay — APPROVED** [review]

**Comment by TooAngel** [comment]
This pull request was merged by [worlddriven](https://www.worlddriven.org).

**Pull request merged**